ic-helper.ps1

```diff
@@ -66,26 +66,46 @@ function Find-GitRepositories {
         [string]$command,
         [switch]$force
     )
+
+ # Check if the path exists
+
+$foundRepository = $false
+
+try {
+    if (-not (Test-Path $path)) {
```

Comment:
Can you explain why do you need `if` clause in `try-catch` block?

Reply:
The if clause was not necessary needed, I found another way using only try/catch. I did not know how to make it work without if clasue, I complicated myself... Hope is it ok now